src/librustc_error_codes/error_codes/E0745.md

```diff
@@ -0,0 +1,21 @@
+Cannot take address of temporary value.
+
+For example, this will not compile
+
+```compile_fail,E0745
+# #![feature(raw_ref_op)]
+fn temp_address() {
+    let ptr = &raw const 2;   // ERROR
+}
+```
+
+To avoid the error, first bind the temporary to something with a more clearly
+defined lifetime.
+
+```ignore
```

Comment:
Why "ignore"?

Reply:
This doesn't currently compile. I'll remove this once the feature is fully implemented.